documentation/sphinx/source/backups.rst

```diff
@@ -524,6 +524,12 @@ The ``start`` command will start a new restore on the specified (or default) tag
 ``--inconsistent-snapshot-only``
   Ignore mutation log files during the restore to speedup the process. Because only range files are restored, this option gives an inconsistent snapshot in most cases and is not recommended to use.
 
+``--user_data``
+  Restore only the user keyspace. This option should NOT be used alongside --system_metadata (below) and CANNOT be used alongside other specified key ranges
+
+``--system_data``
+  Restore only the relevant system keyspace (currently only tenant metadata). This option should NOT be used alongside --user_data (above) and CANNOT be used alongside other specified key ranges
+
```

Comment:
What is the behavior if none of the two options are specified?

Reply:
Then it acts as before and it will try to restore both the user and system (tenant metadata) keys together. But the order in which they are restored is not guaranteed

Reply:
Yeah basically we need to escape characters when specifying the key ranges which is kind of ugly and error prone, so this makes it easier